includes/class-wc-gutenberg-emails-loader.php

```diff
@@ -13,6 +13,42 @@ class WC_Gutenberg_Emails_Loader {
 	 * Constructor.
 	 */
 	public function __construct() {
+		add_action( 'init', array( $this, 'init_post_type' ) );
+		add_action( 'init', array( $this, 'create_emails' ) );
+	}
+
+	/**
+	 * Init post types
+	 */
+	public function init_post_type() {
+		register_post_type(
+			'woocommerce-emails',
+			array(
+				'labels'           => array(
+					'name'          => __( 'WooCommerce Emails', 'woocommerce-gutenberg-emails' ),
+					'singular_name' => __( 'WooCommerce Email', 'woocommerce-gutenberg-emails' ),
+				),
+				'public'           => false,
+				'show_ui'          => true,
+				'show_in_rest'     => true,
+				'delete_with_user' => false,
+				'map_meta_cap'     => true,
+				'capability_type'  => 'post',
+				'capabilities'     => array(
```

Comment:
I'm not sure if this is intentional, but I can't delete or create emails. If I remove the `capabilities` code I can create them, then resetting back to this PR, I can edit that "email". It seems intentional - we only have a set of emails, and we don't want users creating new ones, but then this code looks like it should allow some user that ability (whoever has `delete_woocommerce_emails` & `create_woocommerce_emails`).

Maybe add a comment explaining the intent here?

Reply:
I was just able to switch the email down to a draft, which then let me trash it. So this probably needs more thought overall.

Reply:
Right, it's intentional. I added a comment explaining that (8bcd16d71711f8365a8215f2a40a080ebcaf2f05).

I also created an issue (#14) to handle the draft/trashing issue, it looks like it will require a different solution than using `capabilities`, so I think it's worth doing that in a follow-up.